nextcloudappstore/core/caching.py

```diff
@@ -33,6 +33,13 @@ def app_rating_etag(request: Any, id: str) -> str:
     return create_etag([(AppRating.objects.filter(app__id=id), 'rated_at')])
 
 
+def all_apps_etag(request: Any) -> str:
```

Comment:
this is the same code as apps_etag right?

Reply:
just use apps_etag, it works for both routes

Reply:
🙈 I didn't noticed that - you are right - fixed